.agent-os/instructions/core/import-ai-prompts.md

```
@@ -0,0 +1,304 @@
+
+<extraction_logic>
+  FOR each comment:
+    IF contains "🤖 Prompt for AI Agents":
+      EXTRACT prompt text from code block
+      EXTRACT file path and line numbers
+      EXTRACT comment ID
+      STORE in prompts array
+</extraction_logic>
+
+<prompt_parsing>
+  - File path: Extract from prompt text
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)